tidb-cloud/tidb-cloud-upload-local-files.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Import Local Files to TiDB Cloud
+summary: Learn how to import local files to TiDB Cloud.
+---
+
+# Import Local Files to TiDB Cloud
+
+You can import local files to TiDB Cloud directly. You only need to select a local file, select the target cluster and table, make changes if needed, and then directly import the data into TiDB Cloud.
+
+This feature now supports the following:
+
+- Importing one CSV file within 50 MiB for one task
+- Only Serverless Tier clusters
+
+## Prerequisites
+
+- Prepare the local file to be imported. The file must be in CSV format within 50 MiB.
+- Use [SQL Editor](/develop/dev-guide-tidb-crud-sql.md#explore-sql-with-tidb) to create a table in the target database in advance. The table schema must be consistent with the local file. Alternatively, you can enter the table name in the import task and let TiDB Cloud help you automatically create the target table.
```

Comment:
Pre-creating a table is not a prerequisite.  Here are the prerequisites: 
Since the CSV data can be imported into an existing table or into a new table, if the CSV data will be imported into an existing table, make sure the column order of the CSV file is the same as the column lists of the target table.